app/src/main/java/com/hedvig/app/feature/addressautocompletion/model/DanishAddressInput.kt

```diff
@@ -0,0 +1,42 @@
+package com.hedvig.app.feature.addressautocompletion.model
+
+class DanishAddressInput private constructor(
+    val rawText: String,
+    val selectedDanishAddress: DanishAddress? = null,
+) {
```

Comment:
A class to make it easier to hold the input data. Since we need the member to be able to free-form input stuff, but also keep information about when they may have clicked on some address which is stored in the `selectedDanishAddress`